felix/dataplane/linux/int_dataplane.go

```diff
@@ -765,6 +766,26 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		dp.iptablesMangleTables = append(dp.iptablesMangleTables, mangleTableV6)
 		dp.iptablesFilterTables = append(dp.iptablesFilterTables, filterTableV6)
 
+		if config.RulesConfig.VXLANEnabled {
+			routeTableVXLAN := routetable.New([]string{"^vxlan-v6.calico$"}, 6, true, config.NetlinkTimeout,
+				config.DeviceRouteSourceAddressIPv6, config.DeviceRouteProtocol, true, unix.RT_TABLE_UNSPEC,
+				dp.loopSummarizer, routetable.WithLivenessCB(dp.reportHealth))
+
+			vxlanManager := newVXLANManager(
```

Comment:
nit: should we name this `vxlanManagerV6` or something to that effect to avoid confusion with the v4 `vxlanManager` code? (same with `routeTableVXLAN` above, `routeTableVXLANV6`?)